src/handlers/assign.rs

```diff
@@ -123,8 +135,31 @@ pub(super) async fn handle_input(
     ctx: &Context,
     config: &AssignConfig,
     event: &IssuesEvent,
-    _input: AssignInput,
+    input: AssignInput,
 ) -> anyhow::Result<()> {
+    let assign_command = find_assign_command(ctx, event);
+
+    // Perform assignment when:
+    // - PR was opened normally
+    // - PR was opened as a draft with an explicit r? (but not r? ghost)
+    // - PR was converted from a draft and there are no current assignees
+    let should_assign = match input {
+        AssignInput::Opened => true,
+        AssignInput::OpenedDraft => {
+            // Even if the PR is opened as a draft, we still want to perform assignment if r?
+            // was used. However, historically, r? ghost was meant to mean "do not
+            // perform assignment". So in that case, we skip the assignment and only perform it once
+            // the PR has been marked as being ready for review.
+            assign_command.is_some() && assign_command.as_deref() != Some(GHOST_ACCOUNT)
```

Comment:
nit: maybe a bit clearer
```suggestion
            assign_command.is_some_and(|a| &a != GHOST_ACCOUNT)
```